CONTRIBUTING.md

```diff
@@ -74,23 +90,42 @@ yarn android
 yarn ios
 ```
 
+## Releasing
+
+To bump SDK version, run the following command:
+
+```sh
+yarn run lerna version <version> --ignore-changes --skip-git
+```
+
+This will bump versions in the `lerna.json` and related `package.json` files, they need to be committed after that.
+
+Normally [lerna version](https://github.com/lerna/lerna/tree/main/commands/version#readme) can bump version only for modules which were changed, generate changelog and push changes, but we are not using these capabilities yet.
```

Comment:
```suggestion
Datadog does not support the functionality where the  [lerna version](https://github.com/lerna/lerna/tree/main/commands/version#readme) can bump version only for the changed, then generate a changelog, and push the changes.
```
shouldn't promise future functionality

Reply:
This sentence is only for us (dev team) as only we release new versions of SDK. It just says about release tooling to the other dev team members.

Reply:
I wanted to point that `Lerna` supports more advanced flow and there is a possible room for the improvement, but I think I'll remove this sentence then.